pkg/common/printer.go

```diff
@@ -10,14 +12,18 @@ import (
 func Print(outcomes ...Outcome) {
 	issuesCount := 0
 	foundIssues := false
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 8, 8, 0, ' ', 0)
+
+	defer w.Flush()
```

Comment:
```suggestion
	defer func(){ _ = w.Flush()}()
```

Reply:
Something like this could help:
https://github.com/openservicemesh/osm-health/blob/7c3b835ef90b4b0a4b3e2f0efb5375af5191e5d1/pkg/osm/types_test.go#L60